inst/components/sidebar.scss

```diff
@@ -316,21 +316,31 @@
 }
 
 
-// In the case of page_navbar(nav_panel("Foo", layout_sidebar())), we want the
-// sidebar to be flush with the viewport (just like it will be in the
-// case of page_navbar(sidebar = sidebar()))
 .navbar + .container-fluid {
-  &:has(>.tab-content>.tab-pane.active>.bslib-sidebar-layout) {
+  // When the tab/page is fillable, undo the padding on the container-fluid
+  // (in this case, the user has control over padding/gap)
+  &:has(> .tab-content > .tab-pane.active.html-fill-container) {
     padding-left: 0;
     padding-right: 0;
   }
```

Comment:
Try to get rid of this by identifying this scenario inside `tagQuery()`

Reply:
I decided against this since keeping these styles in `sidebar.scss` will better guarantee that we're not breaking behavior and be easier for PyShiny to adopt.